jsons/Nations.json

```diff
@@ -311,7 +311,9 @@
 		"uniqueName": "Great Expanse",
 		"uniques": [
 			"Gain control over [8] tiles [in this city] <upon founding a city>",
-			"[+15]% Strength <for [All] units> <when fighting in [Friendly Land] tiles>"
+			"[+15]% Strength <for [All] units> <when fighting in [Friendly Land] tiles>",
+			// Start with Pathfinders instead of Warriors
```

Comment:
``` suggestion
			// Start with Pathfinders instead of Warriors. While this works, it could possibly be exploited by buying Warriors before clicking end turn.
```